index.html

```diff
@@ -0,0 +1,63 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <script
+      src="https://kit.fontawesome.com/739fd14f20.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>project-peter-zewdie</title>
+  </head>
+  <body>
+    <section id="navbar">
+      <div class="nav-container">
+        <div class="nav-items"><a href="#">About</a></div>
+        <div class="nav-items"><a href="#">Work</a></div>
+        <div class="nav-items"><a href="#">Contact</a></div>
+      </div>
+    </section>
```

Comment:
- [ ] Kindly use a descriptive title for the page
- [ ] I think it would be better if you use semantic tags `header` and `nav` for the navigation bar